### PR TITLE
Add epoch times

### PR DESCRIPTION
**Pull request by khornberg**
So, I wanted to make an input element take any date possible and normalize it to something.
In my searching I found `moment.parseFormat` which had already completed most of the work. Thanks!
I also wanted the ability to take epoch times.

Example of what I mean
https://jsfiddle.net/b0xmrus8/

### CODE REVIEW

File: moment.parseFormat.js [hunk 1]
```diff
@@ -55,6 +55,9 @@
   var regexYearShort = /\d{2}/;
 
   var regexFillingWords = /\b(at)\b/i;
+  
+  var regexUnixOffset = /\d{13}/;
```
Comment (gr2m) [inline moment.parseFormat.js]: can we rename this to regexUnixMillisecondTimestamp? Also in the comment below (line 78)

File: moment.parseFormat.js [hunk 2]
```diff
@@ -70,6 +73,13 @@
     options = options || {};
     options.preferredOrder = options.preferredOrder || defaultOrder;
 
+    // EPOCH
+
```
Comment (gr2m) [inline moment.parseFormat.js]: I'd leave out the EPOCH comment, it's clear enough what the lines below do

File: tests/moment.parseFormat-test.js [hunk 3]
```diff
@@ -47,4 +47,8 @@ test('GitHub issues', function() {
   equal( moment.parseFormat('2014-09-04T01:20:28.888+02:00'), 'YYYY-MM-DDTHH:mm:ss.SSSZ', '#23 2014-09-04T01:20:28.888+02:00 → YYYY-MM-DDTHH:mm:ss.SSSZ' );
   equal( moment.parseFormat('2014-09-04T01:20:28.888-02:00'), 'YYYY-MM-DDTHH:mm:ss.SSSZ', '#23 2014-09-04T01:20:28.888-02:00 → YYYY-MM-DDTHH:mm:ss.SSSZ' );
   equal( moment.parseFormat('2014-09-04T01:20:28.888Z'), 'YYYY-MM-DDTHH:mm:ss.SSSZ', '#23 2014-02-02T10:11:58.888Z → YYYY-MM-DDTHH:mm:ss.SSSZ' );
+  
+  // https://github.com/gr2m/moment.parseFormat/pull/
```
Comment (gr2m) [inline tests/moment.parseFormat-test.js]: can you change it to https://github.com/gr2m/moment.parseFormat/pull/29

**Comment by gr2m** [comment]
Could you reword the two commits to follow the conventions that [semantic-release](https://www.npmjs.com/package/semantic-release) to do its auto-release dance?

I'd suggest

1. `test: add unix timestamp support`
2. `feat: add unix timestamp support`

I think it's always nice to commit the test, too. Makes it easier to double check if the tests were red, before fixing / adding missing feature.

If that's too much hassle, no worries, I can do it for you :+1:

**Comment by khornberg** [comment]
sure

**Comment by gr2m** [comment]
I'm so sorry I totally missed your update, we'll get it merged soon, promise

**Comment by khornberg** [comment]
No worries. Thanks.

Kyle

**Comment by gr2m** [comment]
released as moment-parseformat@1.1.0 :ship: :package:  :+1:

**Pull request closed without merging**